src/queries/index.ts

```diff
@@ -1,23 +1,18 @@
 import { useQuery } from "@tanstack/react-query";
 import { DirectoryQuery } from "@/interface";
-import { directory } from "@/mock";
+import { directories } from "@/mock";
 
 //TODO: make this async await for actual data fetching
-function getDirectoryList() {
+function getDirectories() {
```

Comment:
if it is referring to a list we can use `directorylist` instead apply it to other.

Reply:
I want to stick with two terms only, directory and directories. using directoryList as well as directories might get confusing later on when the codebase becomes larger.

list - directories
item - directory

Reply:
I see make sense, all good to me.